.test/scripts/Set-K8sConfig.Tests.ps1

```diff
@@ -0,0 +1,19 @@
+
+$RepoRoot = "$PSScriptRoot/../.."
+$ScriptsRoot = "$RepoRoot/scripts"
+$OutRoot = "$PSScriptRoot/../out"
+
+Describe "Set-K8sConfig" {
+  BeforeAll {
+    &"$ScriptsRoot/Set-K8sConfig.ps1" -Path (Split-Path $OutRoot -Parent)
+  }
+  AfterAll {
+    Remove-Item $OutRoot -Recurse -Force
+  }
+  It "Should create a Json for every Dockerfile" {
```

Comment:
It should create a single json with multiple parsed dockerfiles in it.  At least - that was the initial design and I believe that's what's the modified code is doing.

Reply:
fixed